dependencies.gradle

```diff
@@ -1,23 +1,21 @@
 // Add your dependencies here
 
 dependencies {
-    api('com.github.GTNewHorizons:Baubles:1.0.4:dev')
-    api('com.github.GTNewHorizons:Applied-Energistics-2-Unofficial:rv3-beta-554-GTNH:api')
-
+    api('com.github.GTNewHorizons:Baubles-Expanded:2.1.6-GTNH:dev')
     api('thaumcraft:Thaumcraft:1.7.10-4.2.3.5:dev')
 
+    compileOnly('com.github.GTNewHorizons:Applied-Energistics-2-Unofficial:rv3-beta-576-GTNH:api')
```

Comment:
This should still be marked as `implementation` or `api` in the build script since it will be required to compile with Thaumic Tinkerer as a dependency in some conditions